docs/modules/ROOT/pages/faq.adoc

```diff
@@ -0,0 +1,46 @@
+= Android Frequently Asked Questions (FAQ)
+:hardbreaks:
+:oauth2-app-url: https://marketplace.owncloud.com/apps/oauth2
+:android-legacy-central-url: https://central.owncloud.org/t/local-copy-could-not-be-renamed-try-a-different-name/16715/2
+:android-app-tx-url: https://www.transifex.com/owncloud-org/owncloud/android/
+:android-app-beta-url: https://owncloud.com/beta-testing/#android
+:toc: right
+
+Here you can find some of the most frequently asked questions about the ownCloud Android app.
+
+== Usage
+
+=== Compatibility with ownCloud server older than version 10.0
+
+ownCloud Android app 2.9 and later only supports ownCloud server > 10.0. Check {android-legacy-central-url}[this post in the community forum] how to install older versions.
```

Comment:
We stopped supporting old servers in v2.15 according to https://github.com/owncloud/android/releases/tag/oc-android-2.15

BTW, we could also add that minimum android version supported is Lollipop (API 21). What do you think?

Reply:
@abelgardep check https://github.com/owncloud/android/pull/2969/commits/e459c4de534b3eaa43530dcff7e665458b48ed0f